### PR TITLE
Edit user profile page

### PR DESCRIPTION
**Pull request by jwlexi**
#8

### CODE REVIEW

File: front-end/src/components/Footer.jsx [hunk 1]
```diff
@@ -1,37 +1,37 @@
 import React from "react";
-import { Link, withRouter } from "react-router-dom";
+// import { Link, withRouter } from "react-router-dom";
```
Comment (StevenRS00) [inline front-end/src/components/Footer.jsx:2]: Clean this up

File: front-end/src/components/editUserProfile.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                      <div class="form-group">
+                        <label for="name">Name</label>
+                        <input type="email" class="form-control" id="newName" aria-describedby="emailHelp" placeholder="Enter new name"/>
+                        <small id="emailHelp" class="form-text text-muted"></small>
+                      </div>
+                      <div class="form-group">
+                        <label for="password">Password</label>
+                        <input type="password" class="form-control" id="newPassword" placeholder="New password"/>
+                      </div>
+                      <div class = "p-1"></div>
```
Comment (StevenRS00) [inline front-end/src/components/editUserProfile.jsx:25]: Is this empty div necessary?

File: front-end/src/components/editUserProfile.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                      <button type="submit" class="btn btn-primary">Update</button>
+                    </form> 
+            </div>
+            <div class="col"></div>
```
Comment (StevenRS00) [inline front-end/src/components/editUserProfile.jsx:29]: empty div

File: front-end/src/components/editUserProfile.jsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+            <div class="col">
+              <img src="defaultPic.png" alt="Avatar" class = {styles.round}/>
+            </div>
+          </div>
+          <div class="row align-items-end">
```
Comment (StevenRS00) [inline front-end/src/components/editUserProfile.jsx:34]: empty div

**Review by StevenRS00 — CHANGES_REQUESTED** [review]
Some small requests.

replace gitignore with this
`
# Created by https://www.toptal.com/developers/gitignore/api/react,node
# Edit at https://www.toptal.com/developers/gitignore?templates=react,node

### Node ###
# Logs
logs
*.log
npm-debug.log*
yarn-debug.log*
yarn-error.log*
lerna-debug.log*
.pnpm-debug.log*

# Diagnostic reports (https://nodejs.org/api/report.html)
report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json

# Runtime data
pids
*.pid
*.seed
*.pid.lock

# Directory for instrumented libs generated by jscoverage/JSCover
lib-cov

# Coverage directory used by tools like istanbul
coverage
*.lcov

# nyc test coverage
.nyc_output

# Grunt intermediate storage (https://gruntjs.com/creating-plugins#storing-task-files)
.grunt

# Bower dependency directory (https://bower.io/)
bower_components

# node-waf configuration
.lock-wscript

# Compiled binary addons (https://nodejs.org/api/addons.html)
build/Release

# Dependency directories
node_modules/
jspm_packages/

# Snowpack dependency directory (https://snowpack.dev/)
web_modules/

# TypeScript cache
*.tsbuildinfo

# Optional npm cache directory
.npm

# Optional eslint cache
.eslintcache

# Microbundle cache
.rpt2_cache/
.rts2_cache_cjs/
.rts2_cache_es/
.rts2_cache_umd/

# Optional REPL history
.node_repl_history

# Output of 'npm pack'
*.tgz

# Yarn Integrity file
.yarn-integrity

# dotenv environment variables file
.env
.env.test
.env.production

# parcel-bundler cache (https://parceljs.org/)
.cache
.parcel-cache

# Next.js build output
.next
out

# Nuxt.js build / generate output
.nuxt
dist

# Gatsby files
.cache/
# Comment in the public line in if your project uses Gatsby and not Next.js
# https://nextjs.org/blog/next-9-1#public-directory-support
# public

# vuepress build output
.vuepress/dist

# Serverless directories
.serverless/

# FuseBox cache
.fusebox/

# DynamoDB Local files
.dynamodb/

# TernJS port file
.tern-port

# Stores VSCode versions used for testing VSCode extensions
.vscode-test

# yarn v2
.yarn/cache
.yarn/unplugged
.yarn/build-state.yml
.yarn/install-state.gz
.pnp.*

### Node Patch ###
# Serverless Webpack directories
.webpack/

### react ###
.DS_*
**/*.backup.*
**/*.back.*

node_modules

*.sublime*

psd
thumb
sketch

# End of https://www.toptal.com/developers/gitignore/api/react,node`

File: front-end/src/components/editUserProfile.jsx (hunk 2 above)
Comment (jwlexi) [reply front-end/src/components/editUserProfile.jsx:25]: yes, it adds padding

File: front-end/src/components/editUserProfile.jsx (hunk 3 above)
Comment (jwlexi) [reply front-end/src/components/editUserProfile.jsx:29]: grid layout

**Pull request merged**